src/aks-preview/azext_aks_preview/commands.py

```diff
@@ -51,7 +51,6 @@ def load_command_table(self, _):
     with self.command_group('aks', managed_clusters_sdk, client_factory=cf_managed_clusters) as g:
         g.custom_command('kollect', 'aks_kollect')
         g.custom_command('kanalyze', 'aks_kanalyze')
-        g.custom_command('browse', 'aks_browse')
```

Comment:
If deleting this command will cause breaking change to the user, please mark it in the history notes

Reply:
Hi, I suppose user should always have azure-cli installed before using aks-preview and if a command is not implemented in azure-cli-extensions, it will fall back to the implementation in azure-cli.

For this case, the implementation of `aks browse` is basically the same in azure-cli and azure-cli-extensions (actually a little bit more logic in azure-cli), and after removing this command declaration from the extension side, when `aks browse` is executed, the implementation in azure-cli will be used.